### ISSUE TITLE
Failure during autocorrect with Style/NestedTernaryOperator

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#8173]
## Expected behavior

Autocorrect should complete successfully.

## Actual behavior

Fails with error when running autocorrect

## Steps to reproduce the problem

Run rubocop -a against this code:
```ruby
output_path.nil? ?
  root_element :
  output_path.split('.').inject(root_element.respond_to?(:override) ? root_element.override : root_element) { |element, k| element.nil? ? nil : element[k] }
```

Which fails with this error:

```
Inspecting 1 file
An error occurred while Style/NestedTernaryOperator cop was inspecting /Users/tsmith/Desktop/test.rb:3:32.
To see the complete backtrace run rubocop -d.
.

0 files inspected, no offenses detected
undefined method `condition' for #<RuboCop::AST::BlockNode:0x00007fb6bd9f2920>
Did you mean?  conditional?
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/style/nested_ternary_operator.rb:35:in `block in autocorrect'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/cop.rb:36:in `call'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/corrector.rb:63:in `block (2 levels) in rewrite'
/usr/local/lib/ruby/gems/2.7.0/gems/parser-2.7.1.4/lib/parser/source/tree_rewriter.rb:319:in `transaction'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/corrector.rb:62:in `block in rewrite'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/corrector.rb:60:in `each'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/corrector.rb:60:in `rewrite'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/team.rb:180:in `autocorrect_all_cops'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/team.rb:95:in `autocorrect'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/team.rb:130:in `offenses'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cop/team.rb:75:in `inspect_file'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:299:in `inspect_file'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:247:in `block in do_inspection_loop'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:279:in `block in iterate_until_no_changes'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:272:in `loop'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:272:in `iterate_until_no_changes'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:243:in `do_inspection_loop'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:122:in `block in file_offenses'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:146:in `file_offense_cache'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:120:in `file_offenses'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:111:in `process_file'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:89:in `each'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:89:in `reduce'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:89:in `each_inspected_file'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:78:in `inspect_files'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/runner.rb:39:in `run'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli/command.rb:10:in `run'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli/environment.rb:17:in `run'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli.rb:65:in `run_command'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli.rb:72:in `execute_runners'
/Users/tsmith/dev/oss/rubocop/lib/rubocop/cli.rb:41:in `run'
/Users/tsmith/dev/oss/rubocop/exe/rubocop:13:in `block in <top (required)>'
/usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/Users/tsmith/dev/oss/rubocop/exe/rubocop:12:in `<top (required)>'
/usr/local/lib/ruby/gems/2.7.0/bin/rubocop:23:in `load'
/usr/local/lib/ruby/gems/2.7.0/bin/rubocop:23:in `<top (required)>'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `load'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:46:in `block in <top (required)>'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/usr/local/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:34:in `<top (required)>'
/usr/local/opt/ruby/bin/bundle:23:in `load'
/usr/local/opt/ruby/bin/bundle:23:in `<main>'
```

## RuboCop version

master

**Comment by koic** [comment]
I opened #8174. Thank you for early catching up!
